com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -612,9 +612,6 @@ var Module = {
     *     'engine_arguments':
     *         List of arguments (strings) that will be passed to the engine.
     *
-    *     'persistent_storage':
-    *         Boolean toggling the usage of persistent storage.
-    *
```

Comment:
Invalid option, see https://forum.defold.com/t/how-html-flag-persistent-storage-worked/64253/4?u=agulev

It can't be used